vmdb/app/models/ems_kubernetes.rb

```diff
@@ -40,7 +40,7 @@ def self.event_monitor_class
 
   def authentication_check
     # TODO: support real authentication using certificates
-    true
+    [true, ""]
```

Comment:
Is this still the case?  I thought there was a way to validate the Kubernetes "auth".  Maybe not, though...